<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task5Window</class>
 <widget class="QDialog" name="Task5Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEditResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>379</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditClass1Girls">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>379</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonProcess">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Вывести</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditClass2Girls">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>379</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditClass3Girls">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>379</width>
     <height>83</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>83</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditAllGirls">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>379</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
